client/src/sagas/variant.js

```diff
@@ -72,7 +72,7 @@ function* countVariantsForPatient(action) {
 function* selectDefaultStatement() {
   const { profile } = yield select(state => state.user);
 
-  if (profile.defaultStatement != null) {
```

Comment:
`profile.defaultStatement` is an empty string si it was creating an error of selecting the statement with an "empty string"